src/lf_apply.rs

```diff
@@ -13,21 +16,12 @@ use std::ffi::c_int;
 use std::ffi::c_uint;
 use std::ffi::c_void;
 
-pub type pixel = u16;
-
-#[inline]
-unsafe fn PXSTRIDE(x: ptrdiff_t) -> ptrdiff_t {
-    if x & 1 != 0 {
-        unreachable!();
-    }
-    return x >> 1;
-}
-
-unsafe fn backup_lpf(
+// TODO(perl) Temporarily pub until mod is deduplicated
+pub(crate) unsafe fn backup_lpf<BD: BitDepth>(
```

Comment:
This shouldn't be needed anymore, right?  Since the module is entirely deduplicated now.

Reply:
derp, no, good point